libfdt/fdt_ro.c

```diff
@@ -247,6 +247,12 @@ int fdt_subnode_offset(const void *fdt, int parentoffset,
 	return fdt_subnode_offset_namelen(fdt, parentoffset, name, strlen(name));
 }
 
+static const char *fdt_get_symbol_namelen(const void *fdt,
```

Comment:
This seems sufficiently useful that it would be worth making public.

Reply:
OK. Will refactor a bit so it gets added by itself then, and then used in fdt_path_offset_namelen().